backend/testing_integration.py

```
@@ -0,0 +1,596 @@
+        try:
+            conn = await asyncpg.connect(self.database_url)
+            
+            # Check if monitoring tables are accessible
+            monitoring_tables = ["health_check_results", "validation_sessions"]
+            accessible_tables = []
+            
+            for table in monitoring_tables:
+                try:
+                    await conn.fetchval(f'SELECT COUNT(*) FROM "{table}" LIMIT 1')
+                    accessible_tables.append(table)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)